src/web-index/webIndex.ts

```diff
@@ -26,16 +44,14 @@ export function replaceWebIndexStrings(str: string, keys: { [key: string]: any }
   });
 }
 
-export function getEssentialWebIndexParams(config: IWebIndexConfig | boolean) {
+export function getEssentialWebIndexParams(config: IWebIndexConfig | boolean, log: FuseBoxLogAdapter) {
   let templatePath = join(env.FUSE_MODULES, 'web-index-default-template/template.html');
```

Comment:
should we use the contents of this default template instead of the constant you defined earlier?